wPortal/Roles.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Roles
+filename: "Roles.md"
+layout: custom
+keywords: []
+headings: ["Overview", "Custom Roles"]
+links: ["/wPortal/INTERJECT-Roles.html", "/wPortal/FolderPermissions.html"]
+image_dir: "Roles"
+images: [
+{file: "Roles", type: "png", site: "Portal", cat: "Roles", sub: "", report: "", ribbon: "", config: ""},
+{file: "CreateNewRoleButton", type: "png", site: "Portal", cat: "Roles", sub: "", report: "", ribbon: "", config: ""},
+{file: "CreateNewRole", type: "png", site: "Portal", cat: "Roles", sub: "", report: "", ribbon: "", config: ""},
+{file: "EditDelete", type: "png", site: "Portal", cat: "Roles", sub: "", report: "", ribbon: "", config: ""},
+{file: "AssignRole", type: "png", site: "Portal", cat: "Roles", sub: "", report: "", ribbon: "", config: ""},
+]
+description: The Roles page allows you to manage Interject Roles by assigning them to the users in your organization. You can also view how many users currently have a role assigned. You can also add custom roles.
+---
+* * *
+ 
+## Overview
+
+The Roles page allows you to manage [Interject Roles](/wPortal/INTERJECT-Roles.html) by assigning them to the users in your organization. You can also view how many users currently have a role assigned. You can also add custom roles.
+
+![](/images/Roles/Roles.png)
+<br>
+
+### Custom Roles
+
+Custom roles relate to [Folder Permissions](/wPortal/FolderPermissions.html) and give you greater flexibility in controlling folder access in the Report Library. 
+
+**Step 1:** Click on the **Create New Role** button:
+
+![](/images/Roles/CreateNewRoleButton.png)
+
+**Step 2:** Give the role a name and description and click **Create**:
+
+![](/images/Roles/CreateNewRole.png)
+<br>
+
+After the role is created, you can edit it or delete it by clicking on the Actions icons.
+
+<blockquote class=highlight_note>
+<b>Note:</b> You can only delete custom roles and only if they are not assigned to any users or folders.
```

Comment:
it seems like this contradicts the previous bullet, which says you can _edit_ or delete